cdap-ui/app/cdap/components/NamespacesPicker/index.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {getCurrentNamespace} from 'services/NamespaceStore';
+
+function NamespacesPickerView() {
```

Comment:
nit: why not just `export default function NamespacesPickerView()`?

Reply:
there will be an expansion to this, it will be a connected component, it's just unfinished at the moment